src/main/scala/nl/knaw/dans/easy/umd/Command.scala

```diff
@@ -59,8 +59,14 @@ object Command {
     new AutoDetectReader(new FileInputStream(file))
   }
 
-  private def requireUTF8(file: File, reader: AutoDetectReader) = Try {
-    require(reader.getCharset.toString == "UTF-8", s"encoding of $file must be UTF-8 but is ${reader.getCharset}")
+  val acceptedCharsets = Set("UTF-8", "ISO-8859-1")
```

Comment:
@jo-pol , @lindareijnhoudt 
And it doesn't make any sense to me to accept Latin-1. If you actually use Latin-1, while the program expects UTF-8, you can create invalid input. Am I missing your point?